automation/evidence/inspect_pr_marker_counts.py

```
@@ -0,0 +1,140 @@
+    last_automation_comment_time: Optional[str]
+
+
+def _compute(repo: str, pr_number: int) -> MarkerCounts:
+    payload = _gh_pr_view(repo, pr_number)
+    comments = _extract_comments(payload)
+
+    monitor = JleechanorgPRMonitor()
+    head_sha = payload.get("headRefOid")
+
+    workflow_counts = {
+        "pr_automation": monitor._count_workflow_comments(comments, "pr_automation"),
```

Ruff (0.14.10)

Private member accessed: `_count_workflow_comments`

(SLF001)